libs/langchain-community/package.json

```diff
@@ -219,6 +220,7 @@
     "@zilliz/milvus2-sdk-node": ">=2.2.7",
     "better-sqlite3": "^9.4.0",
     "cassandra-driver": "^4.7.2",
+    "cborg": "^4.1.1",
```

Comment:
Could you add all these deps as peer deps and mark them as optional?

https://github.com/langchain-ai/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#third-party-dependencies

Reply:
Fixed: 21a55f1efb45f92f813614c3963c51a97f13138f